experimental/OrthogonalDiscriminants/src/utils.jl

```diff
@@ -0,0 +1,103 @@
+@doc raw"""
+    order_omega_mod_N(d::IntegerUnion, q::IntegerUnion, N::IntegerUnion) -> Pair{Bool, Bool}
+
+Return `(flag_plus, flag_minus)` where `flag_plus` and `flag_minus`
+are `true` or `false`, depending on whether `N` divides the order
+of the orthogonal groups $\Omega^+(d, q)$ and $\Omega^-(d, q)$.
+
+# Examples
+```jldoctest
+julia> Oscar.OrthogonalDiscriminants.order_omega_mod_N(4, 2, 60)
+(false, true)
+
+julia> Oscar.OrthogonalDiscriminants.order_omega_mod_N(4, 5, 60)
+(true, true)
+```
+"""
+function order_omega_mod_N(d::IntegerUnion, q::IntegerUnion, N::IntegerUnion)
+  @req is_even(d) "d must be even"
+  m = div(d, 2)
+  exp = 0
+  while mod(N, q) == 0
+    exp = exp + 1
+    N = div(N, q)
+  end
```

Comment:
Actually, I think it might be possible to replace this by
```suggestion
  exp, N = remove(N, q)
```